resources/test/conftest.py

```diff
@@ -17,8 +17,10 @@ def pytest_addoption(parser):
     parser.addoption("--binary", action="store", default=None, help="path to browser binary")
 
 def pytest_collect_file(path, parent):
+    test_type = os.path.relpath(str(path), HERE).split(os.path.sep)[1]
```

Comment:
I feel a little queasy about this line, but I guess it's fine if you add a comment along the lines of "Tests are organized in directories by type".

Reply:
Got it